internal/workflow_testsuite.go

```diff
@@ -176,13 +177,38 @@ func (t *TestActivityEnvironment) ExecuteLocalActivity(activityFn interface{}, a
 }
 
 // SetWorkerOptions sets the WorkerOptions that will be use by TestActivityEnvironment. TestActivityEnvironment will
-// use options of Identity, MetricsScope and BackgroundActivityContext on the WorkerOptions. Other options are ignored.
+// use options of BackgroundActivityContext, MaxConcurrentSessionExecutionSize, and WorkflowInterceptorChainFactories on the WorkerOptions.
+// Other options are ignored.
 // Note: WorkerOptions is defined in internal package, use public type worker.Options instead.
 func (t *TestActivityEnvironment) SetWorkerOptions(options WorkerOptions) *TestActivityEnvironment {
 	t.impl.setWorkerOptions(options)
 	return t
 }
 
+// SetDataConverter sets data converter.
```

Comment:
What about adding setClientOptions instead of all of these?

Reply:
Yes, it is possible. Not sure if it is better though.

Reply:
I am actually thinking about replacing `SetWorkerOptions` with 3 setters. I believe `ClientOptions` and `WorkerOptions` should be used to create client and worker only.